jujugui/static/gui/src/app/utils/environment-change-set.js

```diff
@@ -1179,11 +1179,11 @@ YUI.add('environment-change-set', function(Y) {
               // Update the service name. The add_unit record is first added
               // passing the initial service name. This service name can be
               // changed by users before the changes are committed.
```

Comment:
a comment here about the ghost -> deployed service ID would probably also be helpful